strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/booters/StorageBooter.java

```diff
@@ -49,8 +46,10 @@
     @Inject
     private PropertiesBooter propertiesBooter;
 
-    private Path lockFile;
+    private ILock lock;
```

Comment:
`private ILock lock` -  we don't have to keep it as an instance field - it can be a local variable in the `initialize` method.

Reply:
Refactored in commit b9c73e8